lib/moab/storage_object_validator.rb

```diff
@@ -93,13 +96,31 @@ def check_correctly_formed_moabs
         after_result_size = errors.size
         # run the following checks if this version dir passes check_sub_dirs, even if some prior version dirs didn't
         if before_result_size == after_result_size
-          data_dir_path = "#{version_path}/#{DATA_DIR_NAME}"
-          data_sub_dirs = sub_dirs(data_dir_path)
-          errors.concat check_sub_dirs(data_sub_dirs, version_dir, EXPECTED_DATA_SUB_DIRS)
-          errors.concat check_required_manifest_files(version_path, version_dir)
+          errors.concat check_expected_data_sub_dirs(version_path, version_dir)
+          if after_result_size == errors.size
+            errors.concat check_metadata_dir_file_only(version_path)
+          end
```

Comment:
I think this erroneously removes "check_required_manifest_files" check, no?   If you put it back, you can do #94 as well ...

Reply:
💯 👍